changes.go

```diff
@@ -223,17 +223,40 @@ type TopicInput struct {
 	Topic string `json:"topic,omitempty"`
 }
 
+type Account struct {
+	AccountId int      `json:"_account_id"`
+	Name      string   `json:"name"`
+	Email     string   `json:"email"`
+	Username  string   `json:"username"`
+	Tags      []string `json:"tags,omitempty"`
+}
+
+type SubmitRecordLabel struct {
+	Label     string   `json:"label"`
+	Status    string   `json:"status"`
+	AppliedBy *Account `json:"applied_by,omitempty"`
```

Comment:
According to the docs, this is an `AccountInfo` type and not an `Account` type, see https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#submit-record-info

We have an `AccountInfo` type at https://github.com/andygrunwald/go-gerrit/blob/0983e87949db2bc5c9b166b499d8741b95db048e/accounts.go#L18